app/models/building.rb

```diff
@@ -0,0 +1,9 @@
+class Building < ActiveRecord::Base
+  belongs_to :neighborhood
+
+  validates_presence_of :street1
+  validates_presence_of :city
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.